src/plugins/myItems/myItemsInterceptor.js

```diff
@@ -37,14 +37,15 @@ function myItemsInterceptor(openmct, identifierObject, name) {
             return identifier.key === MY_ITEMS_KEY;
         },
         invoke: (identifier, object) => {
-            if (openmct.objects.isMissing(object)) {
+            if (!object || openmct.objects.isMissing(object)) {
```

Comment:
My first impression is this looks weird, as a null object surely would also be missing? Though I'm OK just checking for null here too.

Reply:
So I'm glad you pointed this out. This is actually guaranteed to be falsy because we are running the `myItemsInterceptor` prior to running the `missingObjectInterceptor` now. I settled on keeping it this way in case there might be some kind of weird backwards-compatibility edge case?

Otherwise I would rather put the falsy check into `isMissing()`.

Reply:
...which I guess would also work? 🤔